src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -1609,6 +1611,68 @@ public SatisfactionRating createSatisfactionRating(Ticket ticket, SatisfactionRa
         return createSatisfactionRating(ticket.getId(), satisfactionRating);
     }
 
+    //////////////////////////////////////////////////////////////////////
+    // Action methods for Dynamic Content - Items and Variants
+    //////////////////////////////////////////////////////////////////////
+
+    public Iterable<Item> getItems() {
```

Comment:
Little dangerous using getItemX if this is specific to dynamic content as it may clash with another feature.

Reply:
@johnou how do you mean clash? I don't remember seeing this as ambiguous, or anything using reflection that could affect using this.

Reply:
In the sense Zendesk may introduce Article items.. and Item would be ambiguous, why not rename to DynamicContentItem?

Reply:
I checked what Zendesk named it in the official SDK.. https://github.com/zendesk/zendesk_api_client_php/blob/master/src/Zendesk/API/Resources/Core/DynamicContentItems.php

Reply:
@johnou got it, sounds good, I will make this change and push here.